### PR TITLE
Leaderboards and footraces

### PR DESCRIPTION
**Pull request by enteryournamehere**
- Implement leaderboards, including separate weekly/all-time sections.
  - Friend-only leaderboards are not yet implemented as Friends functionality is currently not complete.
- Add footrace script.
- Add Monument race script.
- Add a way to override objects to bypass the render distance filter; without this, the client isn't notified of a change in activity participants if they're out of range. More specifically, the client still thinks you're in a footrace if you end it while far away from the place where you start.

### CODE REVIEW

**Review by MickVermeulen — APPROVED** [review]
Nice work :D

File: Uchu.Core/Serializable/Struct/Property/PacketProperty.cs [hunk 1]
```diff
@@ -31,7 +31,7 @@ public class PacketProperty : IPacketProperty
                         var data = (LegoDataDictionary) o;
                         if (data != null)
                         {
-                            var ldf = data.ToString();
+                            var ldf = data.ToString("\n");
```
Comment (TheNexusAvenger) [inline Uchu.Core/Serializable/Struct/Property/PacketProperty.cs:34]: Did this change have any effects beyond leaderboards? Pretty sure there was another script somewhere that had 2 or more LDF entries.

**Review by TheNexusAvenger — APPROVED** [review]
Thanks for covering this change instead of me having to do with Avant Gardens Survival. Surprised how many "LU moments" appeared with this, but somehow not at the same time.

File: Uchu.Core/Serializable/Struct/Property/PacketProperty.cs (hunk 1 above)
Comment (enteryournamehere) [reply Uchu.Core/Serializable/Struct/Property/PacketProperty.cs:34]: The player list you see when you load into AGS with 2+ players will now also highlight your own name correctly. I don't remember if there were any more places where this made a difference (but it didn't break anything in any case).

**Pull request merged**